[REVIEW] Streamline cluster startup (#125)
rapidsai/tpcx-bb: README.md

In `cluster_configuration/cluster-startup.sh`:

    - Update `TPCX_BB_HOME=...` to location on disk of this repo
    - Update `CONDA_ENV_PATH=...` to refer to your conda environment path.
    - Update `CONDA_ENV_NAME=...` to refer to the name of the conda environment you created, perhaps using the `yml` files provided in this repository.
    - Update `CLUSTER_MODE="NVLINK"` to refer to your communication method, either "TCP" or "NVLINK".
    - You may also need to change the `LOCAL_DIRECTORY` and `WORKER_DIR` depending on your filesystem. Make sure that these point to a location to which you have write access and that `LOCAL_DIRECTORY` is accessible from all nodes.

> To start up the cluster on your scheduler node, please run the following from `tpcx_bb/cluster_configuration/`.

Reviewer: ```suggestion
To start up the cluster on your scheduler node, please run the following from `tpcx_bb/cluster_configuration/`. This will spin up a scheduler and one Dask worker per GPU.
```

```bash
bash cluster-startup.sh SCHEDULER
```

Then run the following on every other node from `tpcx_bb/cluster_configuration/`.

```bash
bash cluster-startup.sh
```

> This will spin up a scheduler and one Dask worker per GPU. If you are running on a single node, you will only need to run `bash cluster-startup.sh SCHEDULER`.

Reviewer: ```suggestion
This will spin up one Dask worker per GPU. If you are running on a single node, you will only need to run `bash cluster-startup.sh SCHEDULER`.
```

python benchmark_runner.py --config_file benchmark_runner/benchmark_config.yaml
```

> By default, this will run each pure Dask query once. You can control the number of repeats by changing the `N_REPEATS` variable in the script.

Reviewer: Confused by this, wont this run both `DASK` and `Blazing` queries `N_REPEATS` times.

Author: Yes, good catch. Needs an update

Author: Resolved by https://github.com/rapidsai/tpcx-bb/pull/125/commits/203277c7fc94aa6ca4073b68e02794542db920db

> By default, this will run each pure Dask query once. You can control the number of repeats by changing the `N_REPEATS` variable in the script.

Author: ```suggestion
By default, this will run each Dask query once, and, if BlazingSQL queries are enabled in `benchmark_config.yaml`, each BlazingSQL query once. You can control the number of repeats by changing the `N_REPEATS` variable in the script.
```
